Analysis/TauFinder/src/TaJetClustering.cc

```diff
@@ -457,12 +463,7 @@ void TaJetClustering::processEvent( LCEvent * evt ) {
     pRemainCol->addElement(p);
   }
   evt->addCollection( pRemainCol, _remainCollectionName );
-  
-  //-- note: this will not be printed if compiled w/o MARLINDEBUG=1 !
-  streamlog_out(DEBUG) << "   processing event: " << evt->getEventNumber() 
```

Comment:
... why did you take this DEBUG printout out - it causes no harm and might be useful in a real debugging ?

Reply:
Mainly because it looked like a leftover from the example on which the processor is based. Additionally, I think it should be at the beginning of `processEvent` rather than at the end to make it really useful. And I didn't find any mention of `MARLINDEBUG` in iLCInstall or the cmake config of Marlin or MarlinReco, so I thought this will not be used in any case.